examples/oval.rb

```diff
@@ -0,0 +1,17 @@
+Shoes.app(
+  title: "Schwad's unbelievable desktop application that renders an oval",
+  height: 700
+) do
+  flow do
+    para "Positional arguments:"
+    oval 30, 30, 80, 200, center: true
+  end
+  flow do
+    para "As a circle"
+    oval 30, 30, 80, center: true
+  end
+  flow do
+    para "Keyword arguments:"
+    oval top: 20, left: 20, height: 160, width: 90, center: true, stroke: red, strokewidth: 5
```

Comment:
<img width="536" alt="image" src="https://github.com/scarpe-team/scarpe/assets/7865030/5ab0bc61-d4dc-43b0-b94c-644d209a06ce">

@noahgibbs I'm really getting there!

But I think I'm missing a piece in how I set up fill, stroke, strokewidth to do this correctly

any thoughts or tips?